alicloud/resource_alicloud_db_instance.go

```diff
@@ -68,14 +68,19 @@ func resourceAlicloudDBInstance() *schema.Resource {
 				ForceNew:     true,
 				Default:      Postpaid,
 			},
-
 			"period": {
 				Type:             schema.TypeInt,
 				ValidateFunc:     validateAllowedIntValue([]int{1, 2, 3, 4, 5, 6, 7, 8, 9, 12, 24, 36}),
 				Optional:         true,
 				Default:          1,
 				DiffSuppressFunc: rdsPostPaidDiffSuppressFunc,
 			},
+			"monitoring_period": {
+				Type:         schema.TypeInt,
+				ValidateFunc: validateAllowedIntValue([]int{5, 60, 300}),
+				Optional:     true,
+				Default:      300,
```

Comment:
Please remove the `Default: 300` and if the system has a default, please set it to `Compute: true`